src/validation.cpp

```diff
@@ -2929,6 +2929,7 @@ bool Chainstate::FlushStateToDisk(
         }
         // Flush best chain related state. This can only be done if the blocks / block index write was also done.
         if (fDoFullFlush && !CoinsTip().GetBestBlock().IsNull()) {
+            if (auto mem_gib{coins_mem_usage >> 30}) LogWarning("Flushing large %d GiB UTXO set to disk...", mem_gib);
```

Comment:
Could you please name a constant for this?

And while we're at it... it'd be nice to express these sizes in terms of user-defined literals. See:
https://github.com/theuni/bitcoin/commits/user-defined-size-literals/

So this could become:
```c++
static constexpr auto WARN_FLUSH_COINS_SIZE = 1GiB;
if(coins_mem_usage >= WARN_FLUSH_COINS_SIZE)
...
```

Reply:
I don't mind user defined literals (it's what we did for hexadecimal conversion as well) - but to be fair, [`32<<20`](https://github.com/bitcoin/bitcoin/blob/master/src/dbwrapper.h#L25) isn't a lot worse than`32_MiB` (but the second may be more descriptive, indeed).
Extracted the constant, let me know how you'd like to apply the user-defined literals - my understanding is that they're not merged yet, right?

Reply:
Yeah, I find `coins_mem_usage / 1_GiB` to be much more obvious in meaning.

But sure, it's out of scope for this PR. I'll open another one for that.